test.js

```diff
@@ -72,8 +73,16 @@ describe('v8flags', function () {
 
   it('should fall back to writing to a temp dir if user home can\'t be found', function (done) {
     eraseHome();
-    var v8flags = require('./');
-    var configfile = path.resolve(os.tmpdir(), v8flags.configfile);
+
+    delete require.cache[require.resolve('./config-path.js')];
+    proxyquire('./config-path.js', {
+      'homedir-polyfill': function () {
+        return null;
+      }
+    });
+
+    const v8flags = requireUncached('./');
```

Comment:
don't need this since we're already deleting from the require.cache, just delete 2 things